### PR TITLE
Update README Bookmarklet

### PR DESCRIPTION
**Pull request by shanyuhai123**
Update Bookmarklet in README. Make the URL recoverable to github.com.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ For browser extensions, see [Third-party Related Projects](https://github.com/co
 Or save the following code snippet as a bookmarklet (GitHub markdown doesn't allow js links, so just copy it into a bookmark).
 
 ```
-javascript: window.location.href = window.location.href.replace('github.com', 'github1s.com')
+javascript: window.location.href = window.location.href.replace(/github(1s)?.com/, function(match, p1) { return p1 ? 'github.com' : 'github1s.com' })
```
Comment (xcv58) [inline README.md:22]: Could you please add an explanation for the behavior to convert back to github.com?

**Review by xcv58 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (shanyuhai123) [reply README.md:22]: Convert back to github.com to view issues and switch branch(Although mentioned in #155, I haven't seen it before).

Comment (xcv58) [reply README.md:22]: I mean add this in the README file, thanks.

**Review by conwnet — APPROVED** [review]
lgtm, thanks

**Pull request merged**